app_customers/lib/pages/dashboard/dashboard_view.dart

```diff
@@ -100,40 +102,59 @@ class _DashboardBody extends StatelessWidget {
     return Scaffold(
       resizeToAvoidBottomInset: false,
       extendBody: true,
-      bottomNavigationBar: CustomAnimatedBottomBar(
-        containerHeight: 70,
-        selectedIndex: navigationShell.currentIndex,
-        showElevation: true,
-        itemCornerRadius: 40,
-        curve: Curves.easeIn,
-        onItemSelected: handlePress,
-        items: pages.map((e) => e.navigationBarItem).toList(),
+      bottomNavigationBar: AnimatedBottomNavigationBar.builder(
+        itemCount: pages.length,
+        tabBuilder: (index, isActive) {
+          final color = isActive ? AppColors.white : AppColors.gray;
+
+          return Column(
+            mainAxisSize: MainAxisSize.min,
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Icon(
+                pages[index].iconData,
+                size: 24,
+                color: color,
+              ),
+              const SizedBox(height: 4),
+              Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 8),
+                child: AutoSizeText(
+                  pages[index].title,
+                  maxLines: 1,
+                  style: TextStyle(color: color),
+                  group: autoSizeGroup,
+                ),
+              ),
+            ],
+          );
+        },
+        height: 80,
+        splashSpeedInMilliseconds: 300,
+        backgroundColor: AppColors.darkBlack,
+        activeIndex: navigationShell.currentIndex,
+        gapLocation: GapLocation.end,
+        notchSmoothness: NotchSmoothness.defaultEdge,
+        onTap: handlePress,
+        shadow: BoxShadow(
+          offset: Offset(0, 1),
+          blurRadius: 12,
+          spreadRadius: 0.5,
+          color: AppColors.black,
+        ),
       ),
-      floatingActionButton: Stack(children: [
-        Container(
-          height: 70,
+      floatingActionButtonLocation: FloatingActionButtonLocation.endDocked,
+      floatingActionButton: FloatingActionButton(
+        backgroundColor: AppColors.darkBlack,
+        child: Icon(
+          Icons.share,
+          color: AppColors.white,
         ),
-        Positioned(
-          top: 0,
-          width: MediaQuery.of(context).size.width,
-          child: ConvexButton.fab(
-            onTap: () => Share.share(
-              'https://play.google.com/store/apps/details?id=com.karibu.transtu.prod',
-            ),
-            backgroundColor: AppColors.darkBlack,
-            icon: Icons.share,
-            color: AppColors.white,
-            border: 0,
-            thickness: 0,
-            sigma: 0,
-            top: 45,
-            size: 52,
-            iconSize: 35,
-          ),
+        onPressed: () => Share.share(
+          'https://play.google.com/store/apps/details?id=com.karibu.transtu.prod',
```

Comment:
Please join a description to the link. it's too  weird to  just share a link without description to anyone.
e.g: 
```
Hello 👋,
I use Transall to buy or renew data bundle. It's fast, secure and user-friendly 
so I wanted to share it with you. 
Follow this link to download 👉 ...
```